plugins/woocommerce/changelog/update-woo-blocks-refact-rating-block

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: update
+
+Rating Filter block: refact block according to the new Product Filters block structure
```

Comment:
So since we're not announcing this change (experimental) we should make this changelog entry a comment type.

Reply:
Like the suggested commit above?

(_screenshot, just in case_)

<img width="304" alt="image" src="https://github.com/user-attachments/assets/29e810f3-5387-4be6-b96f-d6818172c97f">

Reply:
So I meant a changelog like this is a comment type: https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce/changelog/51227-add-delayed-account-creation-send-password-51193